src/components/ShareToken/ShareToken.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+
+import { ReactComponent as ShareIcon } from '../../images/icon-share.svg';
+import { ReactComponent as CopyIcon } from '../../images/icon-copy.svg';
+import './ShareToken.css';
+
+const ShareToken = ({ token }) => {
+  const [showMobileShare, setShowMobileShare] = useState(false);
+
+  function copyToken() {
+    navigator.clipboard.writeText(token).catch((err) => {
+      document.execCommand(token); // possible fallback for older browsers
+      console.log(err);
+    });
+  }
+
+  return (
+    <>
+      <button
+        type="button"
+        onClick={() => setShowMobileShare(!showMobileShare)}
+        className="icon-only-button header__share-toggle-button share-toggle-button"
+        aria-label="Share list"
+      >
+        <ShareIcon aria-hidden="true" focusable="false" />
```

Comment:
I'm assuming this is for desktop?

Reply:
Not following you here. Which thing? This looks like the button that is visible on mobile and allows you to toggle the token share area open and closed. I can tell you that the `aria-hidden="true" focusable="false"` is a pattern I see used for inline SVG icons. You hide them from possibly older screen readers that might get confused, since we already have an aria-label that is meaningful and the icon is just decoration. And you prevent focus because apparently certain crappy browsers allow focus on SVGs, which we don't want—we want that focus on the button, or other actually interactive stuff!